library/Asana/Api/Task.hs

```diff
@@ -135,7 +144,7 @@ data Task = Task
   , tCompleted :: Bool
   , tCompletedAt :: Maybe UTCTime
   , tCreatedAt :: UTCTime
-  , tCustomFields :: [CustomField]
+  , tCustomFields :: CustomFields
```

Comment:
As I understand it, the `CustomFields` newtype is there to produce a specific `ToJSON` encoding. Is that encoding applicable here?

Reply:
`CustomFields` gives the correct `FromJSON` decoding as well. It's applicable here :+1: